packages/react/src/utils/positioner/Popper.tsx

```diff
@@ -99,35 +103,38 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
   // Is a broken dependency and can cause potential bugs, we should rethink this as all other refs
   // in this component.
 
-  const computedModifiers: PopperJS.Modifiers = useDeepMemo(
+  const computedModifiers: Options['modifiers'] = useDeepMemo(
     () =>
       _.merge(
-        /**
-         * This prevents blurrines in chrome, when the coordinates are odd numbers alternative
-         * would be to use `fn` and manipulate the computed style or ask popper to fix it but
-         * since there is presumably only handful of poppers displayed on the page, the
-         * performance impact should be minimal.
-         */
-        { computeStyle: { gpuAcceleration: false } },
-
-        { flip: { padding: 0, flipVariationsByContent: true } },
-        { preventOverflow: { padding: 0 } },
-
-        offset && {
-          offset: { offset: rtl ? applyRtlToOffset(offset, position) : offset },
-          keepTogether: { enabled: false },
-        },
-
+        [
+          /**
+           * This prevents blurrines in chrome, when the coordinates are odd numbers alternative
+           * would be to use `fn` and manipulate the computed style or ask popper to fix it but
+           * since there is presumably only handful of poppers displayed on the page, the
+           * performance impact should be minimal.
+           */
+          {
+            name: 'computeStyles',
+            options: { gpuAcceleration: false },
+          },
```

Comment:
This shouldn't be required anymore. Popper rounds to the nearest suitable subpixel, and in addition detects the DPI and uses 2D transforms instead of 3D (to preserve subpixel anti-aliasing).